tests/common/jormungandr/commands.rs

```diff
@@ -65,7 +68,22 @@ pub fn get_start_jormungandr_as_passive_node_command(
         .arg("--config")
         .arg(config_path.as_os_str())
         .arg("--genesis-block-hash")
-        .arg(&genesis_block_hash);
+        .arg(&genesis_block_hash)
+        .stderr(get_stdio_from_log_file(&log_file_path));
     println!("Running start jormungandr command: {:?}", &command);
     command
 }
+
+#[cfg(target_os = "windows")]
+fn get_stdio_from_log_file(log_file_path: &PathBuf) -> std::process::Stdio {
+    use std::os::windows::io::{FromRawHandle, IntoRawHandle};
+    let file = File::create(log_file_path).expect("couldn't create log file for jormungandr");
+    unsafe { Stdio::from_raw_handle(file.into_raw_handle()) }
+}
+
+#[cfg(not(target_os = "windows"))]
```

Comment:
That should better be `#[cfg(unix)]`. If we come across a platform that is neither Windows nor has `std::os::unix`, we should know we need to create a `get_stdio_from_log_file` for it :)